SpanishFilter/sections/specific.txt

```diff
@@ -2030,3 +2030,5 @@ urlcero.com##.banner
 urlcloud.us##.adsbygoogle
 !+ NOT_OPTIMIZED PLATFORM(ext_ff, ext_opera, ios, ext_android_cb, ext_ublock)
 urlcloud.us##.banner
+||static.draft5.gg/partners/dell/dell-bg-large.jpg$image
+||comparaplano.com.br/app/uploads/2021/01/910x200-16gb.jpg$image
```

Comment:
I don't recommend these rule because it is dangerous.

Reply:
And, If someone edits these to `||comparaplano.com.br/app/uploads/$image`, useful content maybe will not be downloaded.

Reply:
The ad not reproduced on `comparaplano.com.br` with Portuguese / Spanish IP, so, I removed it for now and changed first rule.
Thanks for your participation:)